### ISSUE TITLE
task 105

### ISSUE THREAD

**Issue by Piratepie** [Hidendra/LWC#961]
I get seemingly random reports from the console from the LWC plugin.

`[LWC] Plugin LWC v4.4.0 (b881) (July 11, 2013) generated an exception while executing task 105
java.lang.NoSuchMethodError: org.bukkit.Server.getOnlinePlayers()[Lorg/bukkit/entity/Player;
        at com.griefcraft.util.Metrics.postPlugin(Metrics.java:335) ~[?:?]
        at com.griefcraft.util.Metrics.access$400(Metrics.java:57) ~[?:?]
        at com.griefcraft.util.Metrics$1.run(Metrics.java:223) ~[?:?]
        at org.bukkit.craftbukkit.v1_11_R1.scheduler.CraftTask.run(CraftTask.java:71) ~[spigot.jar:git-Spigot-54ec0b8-4acd0f4]
        at org.bukkit.craftbukkit.v1_11_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52) [spigot.jar:git-Spigot-54ec0b8-4acd0f4]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_101]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_101]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_101]`

**Comment by trainphreak** [comment]
The versions for Minecraft 1.9 and newer can't be used on 1.8 and older,
and the versions for 1.8 and older can't be used on 1.9 and newer. There
was a change to the Bukkit/Spigot API in 1.9 that causes this error for any
plugin that uses the changed API method.

TL;DR get the right version please.

**Comment by Piratepie** [comment]
cool thanks
